src/tracking.js

```diff
@@ -113,6 +113,8 @@ export function setupLogger() {
     // https://www.paypal.com/buttons/smart
     // https://www.paypal.com/us/gifts/
     const isLoadedInFrame = isPayPalDomain() && window.xprops;
+    const sdkLoadTime =
```

Comment:
Nit: If we don't need the specific number type check, we can use this syntax to return a string value or undefined.
```suggestion
    const sdkLoadTime = loadTime && String(loadTime.toString);
```

Reply:
Ahh load time can be zero though, which breaks the typing

Reply:
oh good point. I forgot about falsy zero